plugins/woo-ai/src/custom.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module '@wordpress/notices';
```

Comment:
Was this to fix the linting errors when importing the module?

Reply:
Indeed, is there a better way to resolve this?

Reply:
I think this is better than the way I'd used 😆 (comments to suppress the erro). I believe ideally we want to import the `@types/wordpress__data` package, but in my previous attempts to do so it results in some spurious TS errors that are unrelated to our work. I think that needs to be tackled separately and I'm uncertain of by who.

Reply:
Let's tackle it separately, for sure 👍